packages/node/src/lib/events/events.interface.ts

```diff
@@ -49,6 +49,7 @@ export type ITriggerOverrides = {
     | 'plivo'
     | 'postmark'
     | 'sendgrid'
+    | 'chat'
```

Comment:
We are seeking to introduce stronger typing across out SDKs. This fallback into `object` creates a weak typing for `chat`.

Can we follow a similar pattern as `email` below, creating an `IChatOverrides` interface and listing the necessary properties to expose in a strongly typed way?

Reply:
totally agree,
i've pushed a minor change that adds the expected keys inside template object, however I think we need to add more types, hence we have media sending and interactive messages, it does have lots of variety in it, i'll try to add more soon,
Let me know if the last change is good so I can move on with the other message types